data/framer.go

```diff
@@ -0,0 +1,8 @@
+package data
+
+// Framer is simply an object that can be converted to Grafana data frames.
+// This interface allows us to interact with types that represent datasource objects
+// Without having to convert them to dataframes first
```

Comment:
We've standardized on writing data source in two words, so might as well be consistent also in code docs :)
```suggestion
// This interface allows us to interact with types that represent data source objects
// Without having to convert them to data frames first
```

Reply:
gotya I did not know that!